src-tauri/src/node.rs

```diff
@@ -13,7 +13,6 @@ use sc_service::{
     BasePath, Configuration, DatabaseSource, KeepBlocks, PruningMode, Role, RpcMethods,
     TracingReceiver,
 };
-use sc_tracing::logging::LoggerBuilder;
```

Comment:
So we can remove dependency?

Reply:
it is already deleted

Reply:
Are you sure you're looking at the correct revision? I can still clearly see it in `Cargo.lock`.

Reply:
I just checked out at this branch from scratch. No `sc_tracing` in the repo. Also no `LoggerBuilder` aside from `tauri_plugin_log::LoggerBuilder`. 
I think we are good, but will not merge until you confirm too

Reply:
Some on, open "Files changed" and look at line 27 of `Cargo.toml`...